Added desert 404 template

Ahsan-Ehtesham opened the pull request:
Fixes #18 

# Work Done:

- Added responsive 404 desert template

# Relevant Screenshots/Gifs

![image](https://user-images.githubusercontent.com/52749230/197379820-e2c43f5b-88e0-4e94-89d7-a88d0716428e.png)

Ahsan-Ehtesham commented:
@matteobruni Kindly add hacktoberfest-accepted label after review

Ahsan-Ehtesham commented:
@matteobruni kindly review

matteobruni reviewed (changes requested):
This configuration is a copy-paste of the `ocean` template. As I said in other reviews, I'd like to have different particle configurations, not different CSS

Ahsan-Ehtesham commented:
> This configuration is a copy-paste of the `ocean` template. As I said in other reviews, I'd like to have different particle configurations, not different CSS

Okay sure, I will change particle configurations

Ahsan-Ehtesham commented:
Updated particles configuration plus solved all merge conflicts, ready to merge if accepted

matteobruni reviewed (changes requested):
The index is broken now, please test everything before asking for a review.

Ahsan-Ehtesham commented:
> The index is broken now, please test everything before asking for a review.

Sorry for that, I have updated it, you can check the screenshots
![image](https://user-images.githubusercontent.com/52749230/198270780-76b13621-b0fe-49d7-b7ec-9d9a6261826f.png)
![image](https://user-images.githubusercontent.com/52749230/198270875-9d06364d-8eae-4232-98a2-d50cf4321cfb.png)
The code I fixed in index.html:
![image](https://user-images.githubusercontent.com/52749230/198271186-4f34c26b-ed99-4bbd-bfaf-5a1d4a527f31.png)

matteobruni commented on data/pages.json:
```diff
@@ -103,5 +103,10 @@
+    },
+    {
+        "description": "Desert 404",
+        "imageUrl": "/__screenshots/desert.png?raw=true",
```
The imageUrl format must be like all the other ones

matteobruni replied:
It's correct that the image is not visible locally, it's not in the repository yet. It will work when pushed.

matteobruni replied:
The URL is just a string, I don't think that changing the value is breaking the index.html, maybe there's another error.

Ahsan-Ehtesham replied:
I resolved the merge conflict as well, so now it shall work

matteobruni reviewed (changes requested):
The main index is still not working. Please test the whole website.

Ahsan-Ehtesham commented:
> The main index is still not working. Please test the whole website.

Can you share the screenshots how it is not working, as I have done everything possible at my side

Ahsan-Ehtesham commented:
Creating a new PR

matteobruni commented:
<img width="1512" alt="Screenshot 2022-10-27 alle 19 51 35" src="https://user-images.githubusercontent.com/176620/198362828-83b44659-f784-46a0-ac88-c325777e1b1e.png">

Ahsan-Ehtesham commented:
> <img alt="Screenshot 2022-10-27 alle 19 51 35" width="1512" src="https://user-images.githubusercontent.com/176620/198362828-83b44659-f784-46a0-ac88-c325777e1b1e.png">

#84 is my new updated PR
